1.initial-k-sweep/analysis.sh

```diff
@@ -16,7 +16,7 @@ PARAM_FILE_PREFIX='config/z_parameter_sweep_'
 PMACS_FILE='../config/pmacs_config.tsv'
 PYTHON_PATH='python'
 ALGORITHMS=( 'tybalt' 'adage' )
-DATASETS=( 'TCGA' 'TARGET' 'GTEX' 'GREIN_human')
+DATASETS=( 'GREIN_human')
```

Comment:
We probably don't want to run it for the other three datasets anymore!

Reply:
agreed

Reply:
We should probably delete those from elsewhere in this module as well, such as in vae_pancancer.py and adage_pancancer.py